src/index.html

```diff
@@ -7,6 +7,457 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home"><img src="./images/logo.png" alt="Moyo logo"></a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container container">
+        <div class="card container__card" data-qa="card">
+          <div class="card__container">
```

Comment:
Do you really need so many containers?
```suggestion
    <main class="main">
      <div class="main__card card" data-qa="card">
```